src/main/drivers/flash_m25p16.c

```diff
@@ -180,6 +181,10 @@ static bool m25p16_readIdentification(void)
         geometry.sectors = 256;
         geometry.pagesPerSector = 256;
         break;
+    case JEDEC_ID_WINBOND_W25Q256:
```

Comment:
Isn't this case mergable with MX25L25635E?

Reply:
Done.